models/db/search.go

```diff
@@ -30,6 +30,11 @@ const (
 // eg: "milestone_id=-1" means "find the items without any milestone.
 const NoConditionID int64 = -1
 
+// AnyConditionID means a condition to filter the records which match any id.
+// The inverse of the above NoConditionID
+// eg: "assignee_id=-2" means "find the issues with an assignee"
+const AnyConditionID int64 = -2
```

Comment:
We can't use it. -2 means "Gitea Actions"

Reply:
Actually I think we need to refactor the filter first, to avoid using these unclear consts ....

The really tricky part is that the search engines also depends id=0 for "no assignee". I think we need to figure all and clarify these logics first before introducing new options .....

Reply:
Best I can do is change it to some other arbitrary number that has no other meaning. Refactoring the filter to use a more proper datatype than `int64` is outside the scope of my Go abilities (this being my first ever contact with it).